### DISCUSSION TITLE
Async Job Notification Service

### DISCUSSION THREAD

**Discussion by AllanOricil** [forcedotcom/cli#2742] [Ideas]
context: https://github.com/forcedotcom/salesforcedx-vscode/discussions/5445

Async sf/sfdx commands push the generated UUID to Async Job Notification Service (AJNS). Which can be installed through VS Code.
The post call must be wrapped with a try/catch because AJNS might not be running since not everyone uses VS Code. Moreover, it must have a very small timeout so that it does not hang for too long. 

An alternative solution while Salesforce apis don't support configuring webhooks: https://github.com/forcedotcom/cli/issues/2068

**Comment by cristiand391** [comment]
closing this as I don't think the vscode folks are planning to work on this.
